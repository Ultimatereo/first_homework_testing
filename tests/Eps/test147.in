4
3.13e-05 3.71e-05 1.28e-05 3.6e-06 4.4e-06
2.17e-05 5.3e-06 4.47e-05 4.7e-05 3.93e-05
2.68e-05 3.4e-05 4.66e-05 1.83e-05 3.74e-05
1.13e-05 3.9e-05 4.84e-05 1.71e-05 4.54e-05
