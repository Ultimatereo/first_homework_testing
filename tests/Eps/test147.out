-0.38835
0.132922
0.855596
0.186756
